<!-- Launch file for the default STRANDS monitored navigation state machine, with human help nodes included. To have the speech and screen interfaces running,you also need to do roslaunch strands_ui strands_ui.launch-->
<launch>
    <arg name="machine" default="localhost" />
    <arg name="user" default="" />

    <arg name="config_file" default="$(find monitored_navigation)/config/pause_config.yaml"/>

    <machine name="$(arg machine)" address="$(arg machine)" env-loader="$(optenv ROS_ENV_LOADER)" user="$(arg user)" default="true"/>    
    
    <node name="monitored_nav" pkg="monitored_navigation" type="monitored_nav.py" output="screen" args="$(arg config_file)" ></node>


</launch>
